py3status/modules/online_status.py

```diff
@@ -66,22 +66,21 @@ class Meta:
     def post_config_hook(self):
         self.color_on = self.py3.COLOR_ON or self.py3.COLOR_GOOD
         self.color_off = self.py3.COLOR_OFF or self.py3.COLOR_BAD
+        self.ping_cmd = ['ping', '-c', '1', '-W', str(self.timeout), self.url]
```

Comment:
`str(self.timeout)` did you test this with python2?

Reply:
Yes. It looks okay.
```
1d [chris:~/src/py3status/py3status/modules] online_status-timeout(+2/-1)+ 130 ± python3 online_status.py
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'full_text': '●', 'color': '#00FF00', 'cached_until': 1511399543.0}
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'full_text': '●', 'color': '#00FF00', 'cached_until': 1511399544.0}
^C%
1d [chris:~/src/py3status/py3status/modules] online_status-timeout(+2/-1)+ ± python2 online_status.py
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'color': '#00FF00', 'full_text': u'\u25cf', 'cached_until': 1511399546.0}
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'color': '#00FF00', 'full_text': u'\u25cf', 'cached_until': 1511399547.0}
^C%
```

Reply:
Switch to `%s` anyway?

Reply:
> `1d [chris:~/src/py3status/py3status/modules] online_status-timeout(+2/-1)+ ± python2 online_status.py`

for these changing the prompt to `$` 'user' or `#` 'root' is customary as it makes things more readable and doesn't leak information so 
```
$ python2 online_status.py
...
```

not a huge deal